clientconn.go

```diff
@@ -1110,6 +1110,7 @@ func (cc *ClientConn) ResetConnectBackoff() {
 
 // Close tears down the ClientConn and all underlying connections.
 func (cc *ClientConn) Close() error {
+	var wg sync.WaitGroup
```

Comment:
Style: declare this immediately before it's needed.

Reply:
Done